defaults/main.yml

```diff
@@ -47,4 +47,14 @@ grafana_mongodb_replica_name: CHANGEME
 graylog_mongodb_hosts: []
   # - YOUR_MONGODB_HOST_1:27017
   # - YOUR_MONGODB_HOST_1:27017
-  # - YOUR_MONGODB_HOST_1:27017
\ No newline at end of file
+  # - YOUR_MONGODB_HOST_1:27017
+
+# Offline installation
+graylog_offline_install: false
+graylog_baseurl: ''
+graylog_gpg_key_path: "{{ playbook_dir }}/files/groups/graylog/key"
+graylog_gpg_key: GRAYLOG-GPG
+graylog_yum_repo_template: graylog.repo.j2
+
+# enable in case if you change data path & log path
+graylog_change_path: false
```

Comment:
maybe use `graylog_change_install_path` for better clarity or change the logic to use install path as variable and run related task if that variable is not empty.